test/utils.py

```diff
@@ -1394,3 +1395,10 @@ def get_response_data(response):
     assert 'errors' not in response_json, response_json
 
     return response_json['data']
+
+
+def consume_lines(port, pipe):
```

Comment:
Name of the function makes it very difficult to tell that it does what it does now.

Reply:
Function copy pasted from tarantool projects as is. I dont think that it's convenient to change name now.